graph-sync/overlays/test/buildconfig.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: BuildConfig
+metadata:
+  labels:
+    app: thoth
+    component: graph-sync
+  name: graph-sync-job
+spec:
+  resources:
+    limits:
+      cpu: 2000m
+      memory: 768Mi
+    requests:
+      cpu: 2000m
+      memory: 768Mi
+  output:
+    to:
+      kind: ImageStreamTag
+      name: "graph-sync-job:latest"
```

Comment:
should we push to test?

Reply:
the argocd application wouldnt allow us to use the test tag in graph-sycn workflow , it would overide it to latest.
may be for now it is okay to use latest.

@goern what are your thoughts on this?

Reply:
hmm, or we reconfig the argocd to not self-heal? so that users can change the workflow temporarily?

Reply:
erm, ja :) one or the other... idk :)